qa/multi-cluster-search/src/test/java/org/elasticsearch/search/CCSDuelIT.java

```diff
@@ -813,6 +817,8 @@ private static void assertAggs(SearchResponse response) {
             List<Map<String, Object>> shards = (List <Map<String, Object>>)profile.get("shards");
             for (Map<String, Object> shard : shards) {
                 replaceProfileTime(shard);
+                // Fetch profiling will be significantly different in fan out mode
```

Comment:
Could you explain this part? I'm not totally clear on why it would be different.

Reply:
I'll add an explanation, yeah. The short version is that what minimizing round trips does is that it'll fetch on the remote cluster and pull back the response and then merge *those* responses. In full fan out mode we'll merge the candidates before the fetching.